src/hooks/useTodoItemCardAdd.ts

```diff
@@ -0,0 +1,36 @@
+import { format } from "date-fns";
+import { ITodoItemCard } from "../components/todo-item-card";
+import axios, { AxiosError } from "axios";
+import { useState } from "react";
+
+const useTodoItemCardAdd = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState("");
```

Comment:
Instead of creating two states, you could create one (it will reduce re-rendering counts).

Reply:
Didn't now about it.